vehicles.sql

```diff
@@ -0,0 +1,20 @@
+CREATE TABLE IF NOT EXISTS `player_vehicles` (
+    `id` int(11) NOT NULL AUTO_INCREMENT,
+    `license` varchar(50) NOT NULL,
+    `citizenid` varchar(50) NOT NULL,
+    `vehicle` varchar(50) DEFAULT NULL,
+    `hash` varchar(50) DEFAULT NULL,
```

Comment:
Should hash be required? Wondering if there is a use case for having a vehicle without a model.